install-deps.sh

```diff
@@ -337,7 +323,7 @@ else
             *Focal*)
                 [ ! $NO_BOOST_PKGS ] && install_boost_on_ubuntu focal
                 $with_zbd && install_libzbd_on_ubuntu focal
-                $with_pmem && install_libpmem_on_ubuntu focal
+                $with_pmem && $SUDO apt-get install -y libndctl-dev libdaxctl-dev libpmem-dev libpmemobj-dev
```

Comment:
we discussed on this part the other day. i suggested install the deps using `debian/control`, and i still do.